### PR TITLE
Update release note with last changes made to 1.1.10

### PR DESCRIPTION
**Pull request by baptistemesta**

### CODE REVIEW

File: md/release_notes.md [hunk 1]
```diff
@@ -9,19 +9,22 @@
 * For the Platform Administrator, a _Health check_ page, to make sure the back-end is up and running
 * New REST API extensions available for the developer to get the results of the algorithm (percentage of chances to finish on time) and use it in existing business applications.
 * Security:
-    - https mechanism added
-    - JWT mechanism with of auto-generated token if needed
+    - https activated by default and configured by the installer
+    - JWT mechanism to protected BICI back-end APIs
```
Comment (laurentleseigneur) [inline md/release_notes.md]:
```suggestion
    - JWT mechanism to secure BICI back-end APIs
```

File: md/release_notes.md [hunk 2]
```diff
@@ -9,19 +9,22 @@
...
     - Input validation implemented to protect against malicious JavaScript
 * Translation in French and Spanish
+* Automatic migration of data to 1.1.0 on startup
+* Documentation of rest api extensions is provided in a [OpenAPI v3](https://openapi.tools/) format
```
Comment (laurentleseigneur) [inline md/release_notes.md]: you may add on which URL, since doc has only some markdown generated content

**Review by laurentleseigneur — CHANGES_REQUESTED** [review]
we should explain how to get the openApi documentation

File: md/release_notes.md (hunk 2 above)
Comment (baptistemesta) [inline md/release_notes.md]:
```suggestion
* Documentation of rest api extensions is provided in a [OpenAPI v3](https://openapi.tools/) format ( `api-doc.yml` in the rest api extension zip file )
```

**Review by laurentleseigneur — APPROVED** [review]

**Pull request merged**